assets/js/commento.js

```diff
@@ -215,10 +226,7 @@
 
 
     var _getComments = function() {
-        var data = {
-            "url": document.location
-        };
-        post(_api.get, data, function(reply) {
+        get(_api.get + '?url=' + document.location, function(reply) {
```

Comment:
Are you sure we won't have any issues with URL encoding and stuff?

Reply:
can you redo this test with the url being `commento.html?test=1&url=2`? Pretty sure you need `encodeURIComponent(document.location)`